### DISCUSSION TITLE
APC mini Led Lighting

### DISCUSSION THREAD

**Discussion by NervenfeiN** [helgoboss/helgobox#1217] [General]
My problem is that the Led lighting inensity is too low. How can one set the Led to full brightness? On the other hand, congratulations to you for this tool........ highly appreeciated.
Best regards, 
Wolfgang

**Comment by helgoboss** [comment]
If I remember correctly, the APC mini uses MIDI NOTE ON messages to control the LEDs and different NOTE ON velocities to define the color/brightness. In that case, play with "Source min/max", which will influence the velocity of the NOTE ON messages generated by ReaLearn's feedback engine.

**Comment by andrasf** [comment]
Hi! I think same goes for APC keys. What does "Source min/max" mean?

**Reply by helgoboss** [reply]
Described in the docs: https://docs.helgoboss.org/realearn

**Reply by andrasf** [reply]
Thanks. I experimented a little and I think this is in `compartment-common.luau`. I bought the controller for Playtime and works great. I just thought it could be brighter if it can be done in a simple way, nothing serious. Thanks the tutorials, really interesting possibilities there.
